docs/src/tutorials/resolves.md

```diff
@@ -0,0 +1,564 @@
+```jldoctest quick
+julia> optimize!(model)
+```
+```
+This is Ipopt version 3.14.19, running with linear solver MUMPS 5.8.1.
+
+Number of nonzeros in equality constraint Jacobian...:      965
+Number of nonzeros in inequality constraint Jacobian.:        0
+Number of nonzeros in Lagrangian Hessian.............:      427
+
+Total number of variables............................:      305
+                     variables with only lower bounds:       61
+                variables with lower and upper bounds:      122
+                     variables with only upper bounds:        0
+Total number of equality constraints.................:      285
+Total number of inequality constraints...............:        0
+        inequality constraints with only lower bounds:        0
+   inequality constraints with lower and upper bounds:        0
+        inequality constraints with only upper bounds:        0
+
+iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls
+   0  1.7342046e+01 5.78e+00 1.45e+00 -11.0 0.00e+00    -  0.00e+00 0.00e+00   0
+   1  1.5717415e+01 2.93e-01 6.72e-01 -11.0 2.10e+01    -  8.46e-01 1.00e+00f  1
+   2  3.8283977e+00 2.28e+00 5.69e-01 -11.0 9.30e+01    -  3.94e-01 1.00e+00f  1
+   3  3.8665197e+00 6.46e-03 5.07e-02 -11.0 6.13e+00    -  6.15e-01 1.00e+00h  1
+   4  3.6493980e+00 4.97e-03 9.58e-02 -11.0 4.58e+00    -  9.45e-07 1.00e+00f  1
+   5  1.9537673e+00 9.30e-01 4.58e-02 -11.0 7.46e+01    -  6.12e-02 8.36e-01f  1
+   6  1.9164754e+00 2.85e-02 2.87e-03 -11.0 8.29e+00    -  9.28e-01 1.00e+00h  1
+   7  1.9165162e+00 2.77e-06 2.29e-06 -11.0 1.21e-01    -  9.99e-01 1.00e+00h  1
+   8  1.9165162e+00 3.10e-12 8.33e-14 -11.0 1.24e-04    -  1.00e+00 1.00e+00h  1
+
+Number of Iterations....: 8
+
+                                   (scaled)                 (unscaled)
+Objective...............:   1.9165161630626244e+00    1.9165161630626244e+00
+Dual infeasibility......:   8.3327431368875851e-14    8.3327431368875851e-14
+Constraint violation....:   3.1024072200125374e-12    3.1024072200125374e-12
+Variable bound violation:   3.4981688941115863e-06    3.4981688941115863e-06
+Complementarity.........:   1.3620439999139647e-11    1.3620439999139647e-11
+Overall NLP error.......:   1.3620439999139647e-11    1.3620439999139647e-11
+
+
+Number of objective function evaluations             = 9
+Number of objective gradient evaluations             = 9
+Number of equality constraint evaluations            = 9
+Number of inequality constraint evaluations          = 0
+Number of equality constraint Jacobian evaluations   = 9
+Number of inequality constraint Jacobian evaluations = 0
+Number of Lagrangian Hessian evaluations             = 8
+Total seconds in IPOPT                               = 0.011
+
+EXIT: Optimal Solution Found.
+```
+
+## Model Predictive Control Script 
+The steps outlined in the sections above can be captured in an MPC loop. This is summarized in the script below:
+```julia
```

Comment:
Let's run this code too to ensure it stays up to date, maybe just limit the number of mpc iterations (e.g., 10) to a small amount with a comment next to it to indicate that it is quite small. We could even have it generate a plot of the MPC response.